code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/SmirnovAndrewTest.java

```diff
@@ -0,0 +1,49 @@
+package ru.unn.agile.ComplexNumber.Model;
+
+import org.junit.Test;
+import ru.unn.agile.ComplexNumber.model.ComplexNumber;
+import static org.junit.Assert.*;
+
+public class SmirnovAndrewTest {
+
+    @Test
+    public void isEmptyConstructorEqualsZeroConstructor() {
+
+        ComplexNumber complexNumberWithEmptyConstructor = new ComplexNumber();
```

Comment:
Remove first empty line in each test

Reply:
Don't think that it's a problem, just matter of style :)